e2e/paths/uploadDocuments_test.js

```diff
@@ -0,0 +1,21 @@
+const config = require('../config.js');
+
+Feature('UploadDocuments');
+
+Before((I, caseViewPage) => {
+	I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventDescription);
+	caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
+});
+
+Scenario('Selecting to follow for a document produces a textarea', (I, uploadDocumentsPage) => {
+	I.selectOption(uploadDocumentsPage.fields.socialWorkChronologyStatus, 'To follow');
+	I.fillField(uploadDocumentsPage.fields.socialWorkChronologyReason, 'mock reason');
+	I.continueAndSubmit(config.eventSummary, config.eventDescription);
+	I.see(`updated with event: ${config.applicationActions.uploadDocuments}`);
```

Comment:
That could be wrapped under `I` as `I.seeEventInCaseHistory(config.applicationActions.uploadDocuments)`